yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -378,6 +379,18 @@ export function makeMembershipWitness<N extends number>(size: N, start: number):
   return new MembershipWitness(size, BigInt(start), makeTuple(size, fr, start));
 }
 
+/**
+ * Creates arbitrary/mocked membership witness where the sibling paths is an array of fields in an ascending order starting from `start`.
+ * @param start - The start of the membership witness.
+ * @returns A read request membership witness with additional fields initialized to 0.
```

Comment:
This comment seems to contradict that the sibling path will not contain zero values.